app/controllers/incoming_text_messages_controller.rb

```diff
@@ -1,5 +1,6 @@
 class IncomingTextMessagesController < ApplicationController
   include TwilioRequestable
+  process_twilio_actions
```

Comment:
I think web hooks are weird, and they're so weird they deserve to always be in their own controller, and therefore TwilioRequestable's code can & should be simpler and affect an entire controller at a time.

Web hooks are weird: You can't handle them with a regular `:update` Rails method because of the `put` vs. `post` distinction you have to handle in `routes.rb`. Plus, they bypass the Rails anti-cross-site-request-forgery protection, and they have their own authorization checking which is separate from our session handling.

Web hooks' weirdness merits putting them in their own controller. I think this makes code review easier. In the `OutgoingTextMessagesController`, it's not super-duper clear from reading it that `update` does the Twilio authorization check. By contrast, if there were a `TwilioWebhooksController` or `OutgoingTextMessagesWebhooksController`, it'd be crystal clear that 100% of the actions in there would be webhook-style.

If we did that, we could remove `process_twilio_actions` and have easier to review code.

Due to that, I'm -1 on `process_twilio_actions`, but open to discussing more.